menas/src/test/scala/za/co/absa/enceladus/rest/services/VersionedModelServiceTest.scala

```diff
@@ -23,16 +23,18 @@ import za.co.absa.enceladus.rest.repositories.VersionedMongoRepository
 
 import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, Future}
+import za.co.absa.enceladus.model.menas.audit.Auditable
 
-abstract class VersionedModelServiceTest[C <: VersionedModel] extends BaseServiceTest {
+abstract class VersionedModelServiceTest[C <: VersionedModel with Product with Auditable[C]] extends BaseServiceTest {
 
   val modelRepository: VersionedMongoRepository[C]
   val service: VersionedModelService[C]
 
   val millis100 = Duration(100, TimeUnit.MILLISECONDS)
-
+  val millis200 = Duration(100, TimeUnit.MILLISECONDS)
+  
   private val validName = "validName"
-
+      
```

Comment:
Whitespace added. I think your IDE isn't respecting the `.editoconfig` settings.